frontend/packages/pipelines-plugin/src/components/repository/PipelineRunRepositories.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { PipelineRunKind } from '../../types';
+import { REPOSITORY_FIELDS, repositoryLabels } from './conts';
+
+export type PipelineRunRepositoriesProps = {
+  pipelineRun: PipelineRunKind;
+};
+
+const PipelineRunRepositories: React.FC<PipelineRunRepositoriesProps> = ({ pipelineRun }) => {
+  const { t } = useTranslation();
+  const plrLabels = pipelineRun.metadata.labels;
+  const repoLabel = repositoryLabels[REPOSITORY_FIELDS.REPOSITORY].labels[0];
+  if (!plrLabels || !plrLabels[repoLabel]) return null;
+
+  return (
+    <dl>
+      {Object.keys(repositoryLabels).map((repositoryLabel) => {
+        const labelKey = repositoryLabels[repositoryLabel].labels;
+        if (repositoryLabel === REPOSITORY_FIELDS.REPO_URL) {
+          return plrLabels[labelKey[0]] && plrLabels[labelKey[1]] ? (
+            <>
+              <dt>{t(repositoryLabel)}</dt>
+              <dd>{`${repositoryLabels[repositoryLabel].baseURL}/${plrLabels[labelKey[0]]}/${
+                plrLabels[labelKey[1]]
+              }`}</dd>
+            </>
+          ) : null;
+        }
+        return plrLabels[labelKey[0]] ? (
+          <>
+            <dt>{t(repositoryLabel)}</dt>
```

Comment:
Since you are looping through the object this key is dynamic value, and you cannot translate a dynamic value like this.

Reply:
Using static translations now to better align